array/DNAm/preprocessing/rmarkdownChild/ctCheck.rmd

```diff
@@ -1,50 +1,18 @@
----
-title: "Brain FANS Project Quality Control within Cell Type"
-author: "`r commandArgs(trailingOnly=T)[4]`"
-output: 
-  html_document:
-    toc: true
-    toc_float: true
-    number_sections: true
----
-```{r global_options, include=FALSE}
-knitr::opts_chunk$set(fig.align = "center", warning=FALSE, message=FALSE, echo = FALSE, cache = FALSE)
-```
 
 ```{r setup, include=FALSE}
-args<-commandArgs(trailingOnly = TRUE)

```

Comment:
first chunk name 'setup' is duplicate of QC.rmd chunk name so causes error/crash